guides/v2.1/frontend-dev-guide/templates/template-email.md

```diff
@@ -218,32 +215,32 @@ The styles for emails are split into several different files.
 
 When implementing a custom theme, you should be able to fully customize email templates by copying the `<Magento_Blank_theme_dir>/web/css/source/_email-extend.less` and `<Magento_Blank_theme_dir>/web/css/source/_email-variables.less` files to your custom theme and editing those files.
 
-<h3 id="custom-fonts">Custom fonts</h3>
+### Custom fonts {#custom-fonts}
 Emails inherit the custom fonts that are defined by the {% glossarytooltip b00459e5-a793-44dd-98d5-852ab33fc344 %}frontend{% endglossarytooltip %} theme. The Magento Blank theme uses the **Open Sans** font. Because **Open Sans** is not a standard system font, `@font-face` rules are used to include web fonts.
 
 Here is an overview of how the font structure for emails works:
 
-* <a href="{{ site.mage2100url }}app/design/frontend/Magento/blank/web/css/source/_email-extend.less"><code>&lt;Magento_Blank_theme_dir&gt;/web/css/source/_email-extend.less</code></a> contains the `@import` directive that requests the `email-fonts.css` file.
+* [`&lt;Magento_Blank_theme_dir&gt;/web/css/source/_email-extend.less`]({{ site.mage2100url }}app/design/frontend/Magento/blank/web/css/source/_email-extend.less) contains the `@import` directive that requests the `email-fonts.css` file.
```

Comment:
Use angle brackets instead of HTML escape characters. The escape characters render as plaintext inside backticks.

> [`<Magento_Blank_theme_dir>/web/css/source/_email-extend.less`]({{ site.mage2100url }}app/design/frontend/Magento/blank/web/css/source/_email-extend.less)